### ISSUE TITLE
Tally on OSC trigger

### ISSUE THREAD

**Issue by mauricev78** [CasparCG/Client#154]
i don't know if this is a feature request or a bug report :)

It would be nice if you could see in the client when a scriptline is remote triggert by OSC
the Red block on the end of the line does blink when your on that line.
but when the line is not selected you will never know if something is triggert remotely

grt
Maurice

**Comment by Zebiolo** [comment]
We really need an indication.

The remote triggering feature is not bound to OSC. In the future we may have other protocols triggering events.

**Comment by jesperstarkar** [comment]
Yeah, it would be super nice to just have an indicator pop up and blink once even for items not selected (remote control with /[uid]).

(Another thing: the blink animation is different if you do control/[uid]/play or control/play)

**Comment by Zebiolo** [comment]
> Another thing: the blink animation is different if you do control/[uid]/play or control/play
Is this a question? The blink animation should be the same.
